src/components/ProductDisplay/productdisplay.component.js

```diff
@@ -1,41 +1,61 @@
 import React, { Component } from 'react';
-import data from '../data'
-import Product from '../Product/product.component'
-
+import Product from '../Product/product.component';
+import { withRouter } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { getProductsRequest } from '../../actions/productsActions';
+import ReactLoading from 'react-loading';
+import "./productdisplay.css"
 class ProductDisplay extends Component {
 
     constructor(props) {
         super(props)
         this.state = {
-            pdts: data,
-        }
+            page: 0,
+            limit: 4
```

Comment:
can have default pageSize as 10 for now, as 4 seems very less in screen can add more records in data.json for presentation purposes.